src/js/components/search/filters/awardType/AwardType.jsx

```diff
@@ -46,19 +46,18 @@ const propTypes = {
 };
 
 export default class AwardType extends React.Component {
-
     render() {
         const awardTypes = (
             this.props.awardTypes.map((type, index) =>
-                <PrimaryCheckboxType
+                (<PrimaryCheckboxType
```

Comment:
Are we starting these on new lines after the opening parenthesis?

Reply:
Yes, multiline components now need to be wrapped in parens.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-wrap-multilines.md